README.md

```diff
@@ -56,43 +57,38 @@ Some extensions are not documented. We encourage you to add documentation for th
 Installation
 ============
 
-For most installs, the `pip`-based command works for most people.
-
-For complex installation scenarios, please look at the documentation for
-installing notebook extensions, server extensions, pre/postprocessors, and
-templates at the Jupyter homepage http://www.jupyter.org.
-More information can also be found in the
-[Wiki](https://github.com/ipython-contrib/IPython-notebook-extensions/wiki).
-
-
 The install process has two steps:
 
```

Comment:
I think we should first explain the installation approach, and then describe the steps in detail. How about this:

To install notebook extensions, three steps are required. First, this Python package needs to be installed. Then, the notebook extensions themselves can be copied to the Jupyter data directory.
Finally, the installed notebook extensions can be enabled, either by using built-in Jupyter commands, or more convenient by using the `jupyter_nbextensions_configurator` server extension.

The Python package installation step is necessary to allow painless installation of the extensions togther with additional items like nbconvert templates, pre-/postprocessors, and exporters.